tex/preamble.tex

```diff
@@ -290,10 +288,10 @@
 \DeclareFieldFormat[inbook,thesis]{title}{\mkbibemph{#1}\addperiod} % italic title with period
 \DeclareFieldFormat[article]{title}{#1} % title of journal article is printed as normal text
 \DeclareFieldFormat[article]{volume}{\textbf{#1}\addcolon\space}
-\renewcommand{\mkbibnamefirst}[1]{\textsc{#1}}
-\renewcommand{\mkbibnamelast}[1]{\textsc{#1}}
+\renewcommand{\mkbibnamegiven}[1]{\textsc{#1}}
+\renewcommand{\mkbibnamefamily}[1]{\textsc{#1}}
 \renewcommand{\mkbibnameprefix}[1]{\textsc{#1}}
-\renewcommand{\mkbibnameaffix}[1]{\textsc{#1}}
+\renewcommand{\mkbibnamesuffix}[1]{\textsc{#1}}
```

Comment:
Fix more warnings. (hopefully I don't accidentally break something)

Reply:
should work okay, i have them this way in evan.sty as well